src/pages/inline_maps.mdx

```diff
@@ -35,11 +42,18 @@ import { default as MapLibre } from "@components/MapLibre.astro";
         containerstyle="width: 100%; height: 256px"
     />
 
+    ... and a static map from an image you have composed 
+
+    /* To add static images from a jpg or png file use the syntax below*/
+    /* you must include the reference name that corresponds to the file path you have listed above */
+    <img src={fiskImg.src} alt="image description" />
```

Comment:
i want to write a wrapper around this so they don't have to import first but this is great for now